libs/SalesforceSDKCore/SalesforceSDKCoreTests/SFNativeRestRequestListener.h

```diff
@@ -29,6 +29,7 @@
 @interface SFNativeRestRequestListener : SFSDKTestRequestListener <SFRestDelegate>
 
 @property (nonatomic, strong) SFRestRequest *request;
+@property (nonatomic, assign) NSTimeInterval sleepDuringLoad;
```

Comment:
To slow down the consumption of the response. Without it, the request will get out of the activeRequests array before the refresh completes in the new test below.